server.go

```diff
@@ -950,9 +950,35 @@ func newServer(ctx context.Context, cfg *Config, listenAddrs []net.Addr,
 	if err != nil {
 		return nil, err
 	}
+
+	// TODO(elle): All previously persisted node announcement fields (ie,
+	//  not just LastUpdate) should be consulted here to ensure that we
+	//  aren't overwriting any fields that may have been set during the
+	//  last run of lnd.
```

Comment:
leaving this as a TODO since it is addressed in https://github.com/lightningnetwork/lnd/pull/8825